kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver.h

```diff
@@ -1251,7 +1251,7 @@ class ResidualBasedEliminationBuilderAndSolver
                     #pragma omp atomic
                     r +=  v;
 #else
-                    values_vector[pos] += Alocal(i_local,j);
+                    values_vector[pos] += rALocal(i_local,j);

```

Comment:
I think this should also be changed somewhere else

Reply:
What do you mean?

Reply:
this is the only place where this is changed
I am pretty sure this also needs to be changed elsewhere

Reply:
I am cleaning the class step by step, as #3245 changed much code at the same time

Reply:
ok now I see

Reply:
I think this is a GitHub misbehaviour (I checked the master version and this is correct).